.travis.yml

```diff
@@ -2,13 +2,19 @@ language: ruby
 sudo: false
 before_install:
   - which bundle || gem install bundler
+script:
+  - bundle list
+  - bundle exec rake
 rvm:
   - 2.2.7
   - 2.3.4
   - 2.4.1
   - ruby-head
   - jruby-9.1.8.0
   - jruby-head
+gemfile:
+  - Gemfile
+  - gemfiles/Gemfile.rack-1
```

Comment:
Would it be possible to move this file to the top-level? I.e. to name it `Gemfile.rack-1.x` or something in the top-level folder instead. Makes it easier to find it that way.